src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <Description>Provides management capabilities for Azure PowerBI Dedicated accounts.</Description>
+    <AssemblyTitle>Microsoft Azure PowerBI Dedicated Management Library</AssemblyTitle>
+    <VersionPrefix>2.0.1</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.PowerBIDedicated</AssemblyName>
+    <PackageId>Microsoft.Azure.Management.PowerBIDedicated</PackageId>
+    <PackageTags>Microsoft Azure resource management;asazure;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```

Comment:
@selasagi also add PackageReleaseNotes for your nuget package.
As this is your first version for this brand new service, start with 0.9.0-preview
Any questions reach out to me.